android/src/main/java/io/invertase/googleads/ReactNativeGoogleAdsAppOpenModule.java

```diff
@@ -0,0 +1,158 @@
+package io.invertase.googleads;
```

Comment:
:scream: oh no, the rename before I merged this. I should have done that the other way around, I am really really sorry. I'll note what I see that I know has changed though
```suggestion
package io.invertase.googlemobileads;
```